sql/types/json.go

```diff
@@ -67,6 +68,24 @@ func (t JsonType) Convert(v interface{}) (doc interface{}, inRange sql.ConvertIn
 		if err != nil {
 			return nil, sql.OutOfRange, sql.ErrInvalidJson.New(err.Error())
 		}
+	case int8:
```

Comment:
floats?

Reply:
I don't think a float case here matters for correctness, because in the default case floats and doubles will both get serialized and then parsed back as doubles, which is the correct behavior.

But adding a case here avoids doing more unnecessary serialization though, so I added it.

Since the observed behavior should be the same, I didn't add a test.